tools/osbuild-image-test

```diff
@@ -413,6 +418,12 @@ def main():
         type=os.path.abspath,
         default="/var/lib/osbuild/out",
         help="directory where result objects are stored")
+    parser.add_argument(
+        "--imi-output",
+        metavar="DIRECTORY",
+        type=os.path.abspath,
+        default="imis",
```

Comment:
Similar comment as above.  `imis` as a directory name isn't really saving us anything.  Better to be clear.